pages/passbook.js

```diff
@@ -0,0 +1,12 @@
+import TransactionHistory from '../components/passbook';
+import TransactionData from '../mock/passbookData.json';
+import { Footer } from '../components/footer';
+
+export default function Passbook() {
+  return (
+    <div>
+      <TransactionHistory transactions={TransactionData} />
+      <Footer />
```

Comment:
What is the use of Footer here? Or is it already present?
Passbook is a component right not a complete page?

Reply:
@Harshi7016 Thanks for taking time to go through my code 👍🏻 
The file which you have reviewed is the /passbook page (crypto.realdevsquad.com/passbook) and it contains my passbook component (which I've called TransactionHistory as of now) and the footer (as it is present on all the pages of all RDS pages).

Thank you!